packages/cashier/src/Stores/Cashier/on-ramp-store.js

```diff
@@ -69,7 +69,9 @@ export default class OnRampStore extends BaseStore {
         } else if (this.selected_provider) {
             if (this.should_show_dialog) {
                 return this.api_error
-                    ? localize('Our server cannot retrieve an address.')
+                    ? localize(
```

Comment:
Is there a refresh option in on-ramp popup modal to go with this content?

Reply:
My bad, when searched for special error text in repo - found just that file, did that before pulling actual data, forgot to return previous text when found another file with that error, thx for your attention!